assets/src/pages/room/components/StreamPlayer/MediaPlayerPeersSection.tsx

```diff
@@ -148,11 +160,11 @@ const MediaPlayerPeersSection: FC<Props> = ({
             video={video}
             audioStream={audio?.stream}
             className={!oneColumn ? clsx(gridConfig.span, gridConfig.tileClass) : undefined}
+            cameraOffImage={showDisabledIcon(video) ? <InitialsImage initials={config.initials} /> : null}
```

Comment:
I think we can simplify it. Take a look at: https://github.com/membraneframework/membrane_videoroom/pull/96/files